### PR TITLE
Finished my ✨ Top 25 Cryptocurrencys Page! ✨ 

### PR DESCRIPTION
**Pull request by bejanaf**
- made a UseEffect + fetch from www.coingecko.com
- new routing to TopCoins.js
- newState to save Data from the Top 25 Cryptocurrency
- created the TopCoins.js and mapped the Data
- add a function to shorten big Digits of Market Cap and Volume
- refactored from TopCoins.js to CoinsCard.js as component

### CODE REVIEW

File: client/src/components/CoinsCard.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components';
+
+export default function Coinscard({ topCoins }) {
+  function ShortenLargeNumbers(num) {
```
Comment (SharineAnkenbrand) [inline client/src/components/CoinsCard.js:4]: That's a great function ✨

File: client/src/components/CoinsCard.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        </CoinWrapper>
+      ))}
+    </>
+  );
+}
+
+const CoinWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
```
Comment (SharineAnkenbrand) [inline client/src/components/CoinsCard.js]: You could consider to sort it alphabetically 😃

File: client/src/components/CoinsCard.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  padding-left: 0.6rem;
+  margin-top: 0.3rem;
+  width: 33vw;
+`;
+
+const CoinImage = styled.img`
+  width: 1.1rem;
+  height: 1.1rem;
+  background-color: white;
+  border-radius: 50%;
```
Comment (SharineAnkenbrand) [inline client/src/components/CoinsCard.js:69]: You could consider to sort it alphabetically 😃

File: client/src/components/CoinsCard.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+`;
+
+const NegativePrice = styled.span`
+  color: red;
+`;
+
+const CoinPricePercent = styled.span`
+  display: flex;
+  font-size: 0.7rem;
+  height: 1.344rem;
```
Comment (SharineAnkenbrand) [inline client/src/components/CoinsCard.js:97]: You could consider to sort it alphabetically 😃

File: client/src/components/CoinsCard.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+const CoinCapVolume = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const CoinMarketCapVolume = styled.span`
+  display: flex;
+  font-size: 0.7rem;
+  height: 1.344rem;
+  align-items: flex-end;
```
Comment (SharineAnkenbrand) [inline client/src/components/CoinsCard.js]: I don't want to repeat myself BUT you could consider to sort it alphabetically 😃 ✨

**Review by SharineAnkenbrand — APPROVED** [review]
All in all, it looks good to me but consider my consider my comments ✨ 😊

File: client/src/components/CoinsCard.js (hunk 2 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js]: fixed :)

File: client/src/components/CoinsCard.js (hunk 3 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js:69]: fixed :)

File: client/src/components/CoinsCard.js (hunk 4 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js:97]: fixed :)

File: client/src/components/CoinsCard.js (hunk 5 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js]: fixed :)

File: client/src/components/CoinsCard.js (hunk 1 above)
Comment (bejanaf) [reply client/src/components/CoinsCard.js:4]: ✨ Thank you :)

**Comment by bejanaf** [comment]
Thank you very much for your code-review, I've solved all of the small improvements mentioned ✨  :)

**Pull request merged**